app/static/js/controllers/accountuserController.js

```diff
@@ -26,6 +26,52 @@ angular.module('uluwatuControllers').controller('accountuserController', ['$scop
             });
         }
 
+        $scope.contains = function(str, suffix) {
+            if (str != undefined) {
```

Comment:
Sould be better in one expression like return str != undefined && str.indexOf(suffix) !== -1